Client/src/components/Shared/AdCard/AdCard.jsx

```
@@ -0,0 +1,154 @@
+      aria-label={`View details for ${ad.title}`}
+    >
+      {showVideo ? (
+        <div className="relative pt-[56.25%]"> {/* 16:9 aspect ratio */}
+          <iframe 
+            className="absolute inset-0 w-full h-full"
+            src={ad.videoUrl}
+            title={ad.title}
+            style={{ border: 0 }}
+            allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" 
+            allowFullScreen
+          ></iframe>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)